<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLiquidSwitch</class>
 <widget class="QWidget" name="FormLiquidSwitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>900</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/layout2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_HC2301">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>204</y>
     <width>146</width>
     <height>146</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/svalve1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_HC2401">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>544</y>
     <width>146</width>
     <height>146</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/svalve1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_HC2303">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>181</y>
     <width>146</width>
     <height>146</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/svalve2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_HC2403">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>446</y>
     <width>146</width>
     <height>146</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/svalve2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_HC2304">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>180</y>
     <width>148</width>
     <height>148</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/svalve8.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_HC2404">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>446</y>
     <width>148</width>
     <height>148</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/svalve8.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_HC2302">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>525</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/svalve5.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2101">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>60</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2102">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>180</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2103">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>300</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2104">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>420</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2201">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2202">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2203">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WIA2204">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>760</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_FICA2305">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>140</y>
     <width>65</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/water_pump.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_FICA2405">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>406</y>
     <width>65</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/water_pump.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_FICA2305">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>140</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_FICA2405">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>406</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_HC2301">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>HC2301</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_HC2302">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>610</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>HC2302</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_HC2303">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>310</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>HC2303</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_HC2304">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>310</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>HC2304</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_HC2401">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>670</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>HC2401</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_HC2403">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>580</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>HC2403</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_HC2404">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>580</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>HC2404</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_FICA2305">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>180</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>FICA2305</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_FICA2405">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>450</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>FICA2405</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2101">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2101</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2102">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2102</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2103">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2103</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2104">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2104</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2201">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2201</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2202">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2202</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2203">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2203</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_WIA2204">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>830</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>WIA2204</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
